admin/pull-readme

```diff
@@ -6,8 +6,8 @@ set -e
 awk 'BEGIN { take=1 }/# Approach/{take=0;print}take' < README.md > README.md.new
 grep '^//!' rustls/src/lib.rs | \
        sed -e 's@^\/\/\! *@@g' | \
-       sed -e 's@](manual::_04_features)@](https://docs.rs/rustls/latest/rustls/manual/_04_features/index.html)@' | \
-       sed -e 's@\[`crypto::CryptoProvider`\]: crate::crypto::CryptoProvider@[`crypto::CryptoProvider`]: https://docs.rs/rustls/latest/rustls/crypto/struct.CryptoProvider.html@' | \
+       sed -e 's@](manual::_04_features)@](https://docs.rs/portable-rustls/latest/portable_rustls/manual/_04_features/index.html)@' | \
+       sed -e 's@\[`crypto::CryptoProvider`\]: crate::crypto::CryptoProvider@[`crypto::CryptoProvider`]: https://docs.rs/portable-rustls/latest/portable_rustls/crypto/struct.CryptoProvider.html@' | \
```

Comment:
I think it would be ideal to use comments for markers rather than specific text with links like these, in a similar fashion to cargo-sync-readme. As I said in #37 I would like to contribute this kind of update to upstream rustls.